merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/metamodel/EffectModelRecord.java

```diff
@@ -0,0 +1,6 @@
+package gov.nasa.jpl.aerie.merlin.processor.metamodel;
+
+import gov.nasa.jpl.aerie.merlin.framework.annotations.ActivityType;
+
+public record EffectModelRecord(String methodName, ActivityType.Executor executor) {
```

Comment:
Why not just call it `EffectModel`?

Reply:
Another example is `ActivityType`, we already have 3 distinct `ActivityType`s:
```
./merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/models/ActivityType.java
./scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/ActivityType.java
./merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/annotations/ActivityType.java
```
Which is okay but when possible it's nice to have a unique name.

Reply:
I agree with @pcrosemurgy. For context, I think the term "record" was used by analogy with databases. as the annotation processor essentially builds up a knowledge base _about_ the model, then runs over all of the records it has collected (into tables!) and acts on that knowledge.

Reply:
My main concern was indeed that we were using it simply because the class is a record, but I can jive with it being more representative of the fact that it's a record of something without being the actual something.